backend/app/api/endpoints/cloud_projects.py

```
@@ -453,6 +454,22 @@ def access_cloud_file(
     )
 
 
+@router.get("/files/{file_id}/content")
+def read_cloud_file(
+    file_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)